app/views/index.erb

```diff
@@ -10,7 +10,13 @@
 <body>
   <div class="main__container">
     <h1>Chitter</h1>
-
+    
+      <% if @user %>
```

Comment:
Would it be a good idea to allow non-users to see peeps or better to only allow users to see peeps?